docker_templates/templates/docker_images_ros2/nightly/create_ros_image.Dockerfile.em

```diff
@@ -67,10 +68,14 @@ ARG ROS2_BINARY_URL=@ros2_binary_url
 RUN wget -q $ROS2_BINARY_URL -O - | \
     tar -xj --strip-components=1 -C /opt/ros/$ROS_DISTRO
 
-# install setup files
-RUN apt-get update && apt-get install -q -y \
-    ros-$ROS_DISTRO-ros-workspace \
-    && rm -rf /var/lib/apt/lists/*
+# Overwrite setup scripts with ones that point to /opt/ros/$ROS_DISTRO
+RUN mkdir -p /tmp/rosws \
+ && cd /tmp/rosws \
+ && git clone --depth 1 https://github.com/ros2/ros_workspace.git -b latest \
+ && COLCON_CURRENT_PREFIX=/opt/ros/$ROS_DISTRO . /opt/ros/$ROS_DISTRO/local_setup.sh \
+ && cmake -DCMAKE_INSTALL_PREFIX=/opt/ros/$ROS_DISTRO ./ros_workspace \
```

Comment:
While this work it is fairly uncommon to have the build directory be a parent of the sources. (If the package decides to write stuff under `ros_workspace` in the build directory that might break.) Therefore I would suggest to:
* `cd /tmp`
* `git clone worl_workspace`
* `mkdir /tmp/ros_workspace/build`
* `cd /tmp/ros_workspace/build`
* `cmake ..`

Reply:
Added `build` directory adjacent to source dir in 4d78e21